lib/src/helpers/extensions.dart

```diff
@@ -99,5 +99,13 @@ extension StorageGlue on Storage {
 }
 
 extension TouchListConvert on TouchList {
-  List<Touch> toList() => TouchListWrapper(this);
+  List<Touch> toList() => JSListWrapper<Touch>(this);
+}
+
+extension HTMLCollectionConvert on HTMLCollection {
+  List<Element> toList() => JSListWrapper<Element>(this);
+}
+
+extension NodeListConvert on NodeList {
+  List<Node> toList() => JSListWrapper<Node>(this);
 }
```

Comment:
Could be a nice idea. If we had to do that, any ideas on how we might introduce these to users? and make them aware abt these new additions?

Reply:
Yeah I think discoverability is a fair question. I do like the idea of a FAQ and have been considering adding one to this package. This could go in there, as well as the ability to work with cross-origin windows, streams, etc. that we expose through helpers. Also helps if I answer the existing bugs pointing users to use this new class instead. :)

Reply:
Sounds good.